doc/platformsh/INSTALL.md

```diff
@@ -35,12 +35,12 @@
    1. Optional: Set the SYMFONY_ENV environment variable to 'prod' or 'dev':  
       `platform project:variable:set env:symfony_env prod`  
       If you don't do this, remote builds will default to 'prod'.
-1. Checkout the v1.9.0 tag (once it is released) as a new branch:  
-   `git checkout -b ezplatform-ee-v1.9.0 v1.9.0`  
+1. Checkout the v1.9.1 tag (once it is released) as a new branch:  
+   `git checkout -b ezplatform-ee-v1.9.1 v1.9.1`  
```

Comment:
Yes... I could leave out the version number entirely and just say "a stable tag", "...since this feature was added", something like that.